src/support/FabricationDefinition.ts

```diff
@@ -44,68 +44,76 @@ Review Notes:
 Review Notes (Dyoung - 2021.01.11)
 	Changed FileLocator() function to Constructor
 	Moved FileLocator object after File object to maintain alphabetical order of objects.
-	Added RUN and INCLUDE to SPECIALTYPES KEywork Group
+	Added RUN and INCLUDE to SPECIALTYPES Keyword Group
 	Minor updates/misspelling in documentation in various locations
-	
+Review Notes (DYoung = 2021.01.25)
+	Added 'BREAKPOINT' to SPEACIALTYPES
+	Added 'OUTPUT()' to Global Functions
+	Edited Description for 'ERROR' Global Function
+	Added 11 'Add____()' and 11 'Remove____()' Methods to 'PRODUCTENTRY' Object.
+	Added 'SEEKENTY()' and 'SEEKLINE()' Methods to 'FILE' Object.
+	Added 'ADDLINK()', 'DELETELINK()' and 'SETDONOTCUTFLAG()' Methods to 'ITEMSTRUCT' Object.
 */
 
 // IMPORTANT: Anywhere (in code) you are representing an actual value of a string, please use single quotes. This does not apply to the /** documentation areas */
 
 
 // This namespace represents a template of the entire Autodesk Fabrication scripting environment
 // There are some caveats to this for OPERATORS, NUMBERS & STRINGS
-export namespace FABRICATION {
+namespace FABRICATION {
```

Comment:
put export back on

Reply:
Note to self: completed locally. Resolve after update committed.